patterns/roks/main.tf

```diff
@@ -15,6 +15,17 @@ provider "ibm" {
 # Landing Zone
 ##############################################################################
 
+data "ibm_is_ssh_keys" "existing_keys" {}
+
+locals {
```

Comment:
Could we avoid the duplication of logic /code across the 3 patterns here. Perhaps put that code in the ssh_key submodule

Reply:
@vburckhardt - This requires some time to investigate and put the logic at a central place, for instance passing the id of existing key if logic is added in ssh_key submodule needs more time. 
Can we create a separate issue to track this change and if possible we can proceed with merging the changes?

Reply:
Sure - I have created the [issue-4483](https://github.ibm.com/GoldenEye/issues/issues/4483) to track this change. 
Also, @vburckhardt Is it possible to get this PR merged if the changes looks fine to you?

Reply:
This is now taken care with the latest changes. 
PR is raised with the changes and these changes no more exists hence no duplication is present now.